leo/cli/commands/query/mod.rs

```diff
@@ -103,18 +104,21 @@ impl Command for Query {
             .call()
             .map_err(|err| UtilError::failed_to_retrieve_from_endpoint(err, Default::default()))?;
         if response.status() == 200 {
-            tracing::info!("✅ Successfully retrieved data from '{url}'.\n");
             // Unescape the newlines.
-            println!("{}\n", response.into_string().unwrap().replace("\\n", "\n"));
-            Ok(())
+            let result = response.into_string().unwrap().replace("\\n", "\n").replace('\"', "");
+            if !recursive {
```

Comment:
Is there a cleaner way to suppress log outputs in the case of one command calling another? In this case `leo execute` and `leo deploy` use `leo query` for checking public balance

Reply:
My gut tells me that you'd want to decide in the caller, not the callee.
Perhaps we change the interface of command to include a method called `display` which would actually print the output of the command. 
I think the current solution is find, can you make a TODO with enough context so that we can revisit later?